docs/android/getstarted.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Android Get Started"
+sidebar_label: "Get Started"
+---
+
+## Getting Started with Home Assistant Android Development
+
+Welcome to the Home Assistant Android development guide! This document will help you set up your environment, fork the repository, and build your first app.
+
+## Setting Up Your Development Environment
+
+To get started, install the latest stable version of [Android Studio](https://developer.android.com/studio). This is the only tool you need to build the applications.
+
+## Fork, Clone, and Create a Branch
+
+### Fork the Repository
+
+1. Open the [Home Assistant Android repository](https://github.com/home-assistant/android).
+2. Click **Fork** to create your own copy of the repository.
+
+> 💡 **Tip:** If you encounter any issues, refer to the [GitHub documentation on forking a repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo).
+
+### Clone Your Forked Repository
+
+Once you've forked the repository, clone it to your local machine using the following command:
+
+```bash
+git clone https://github.com/<your-github-username>/android.git
+```
+
+Alternatively, you can use Android Studio:
+
+1. Go to `File -> New -> Project from Version Control...`.
+2. Enter your repository URL and clone it.
+
+### Create a Branch
+
+Before making any changes, create a new branch with a meaningful name that reflects the work you are doing. For example:
+
+```bash
+git checkout -b feature/add-new-feature
+```
+
+> 💡 **Tip:** If you're new to Git, check out the [Git Branching Guide](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging). You can also create branches directly in Android Studio.
+
+## Build Your First App
+
+Once you have the repository cloned locally, you can build the app using Android Studio or the terminal.
+
+### From Android Studio
+
+1. Open the project in Android Studio.
+2. Sync the Gradle files.
+3. Click the green **Play** button in the top bar. Android Studio will automatically create an emulator and run the app for you.
+
+### From the Terminal
+
+:::info
+You will need the `JAVA_HOME` environment variable set to a JDK. We are currently using the JDK 21.
+:::
+
+#### On macOS/Linux
+
+```bash
+./gradlew assembleDebug
+```
+
+#### On Windows
+
+```powershell
+gradlew.bat assembleDebug
+```
+
+## Firebase Setup
+
+Firebase is used for notifications and distributing builds. If you don't need these features, you can use a mocked Firebase configuration.
```

Comment:
New builds are still pushed to Firebase but it isn't actively used, end users have the public beta track on Google Play. I think @dshokouhi knows the history behind this one? Either way I don't think it's worth a mention here.

Reply:
There was a limit for 200 users, this was actually the first beta for the app before the play store beta was introduced. I believe we have some users still updating from there lol. Personally switched to the internal track as its easier.